api/actions/urls.py

```diff
@@ -6,5 +6,6 @@
 
 urlpatterns = [
     url(r'^reviews/$', views.ReviewActionListCreate.as_view(), name=views.ReviewActionListCreate.view_name),
+    url(r'^requests/$', views.NodeRequestActionCreate.as_view(), name=views.NodeRequestActionCreate.view_name),
```

Comment:
The only other question I had (this really is just a one off, not a whole review) is this requests actions url  
 is `NodeRequestAction` specific -- will there ever be a case where the `/requests/` endpoint will ever be used to create anything but NodeActions? Would it make sense to make this generic now or when the time comes?

Reply:
See [this comment thread](https://github.com/CenterForOpenScience/osf.io/pull/7919#discussion_r161871794). With `django-guardian` on the horizon, we'll be able to grant permissions in a more granular way, with `*Request` objects driving their allocation. It's somewhat ephemeral at this time, I admit, but it doesn't make sense to not do a little future-proofing now. At worst, it gets rewritten (as it would need to be anyway), and at best, it's extensible in a way that saves us a lot of headaches down the road 🐼